spot-client/src/spot-remote/analytics/middleware.js

```diff
@@ -15,6 +52,31 @@ MiddlewareRegistry.register(() => next => action => {
     return result;
 });
 
+/**
+ * Tracks the {@link REMOTE_CONTROL_REQUEST_STATE} action and logs analytics when the PENDING state
+ * is set (it corresponds to the user triggering an action).
+ *
+ * @param {string} requestType - The type of request like audio mute, video mute etc.
+ * @param {string} requestState - The request state.
+ * @param {any} expectedState - Depends on the request type.
+ * @private
+ * @returns {void}
+ */
+function _remoteControlRequestState({ requestType, requestState, expectedState }) {
+    if (requestState !== requestStates.PENDING) {
+        return;
+    }
+
+    switch (requestType) {
+    case requestTypes.AUDIO_MUTE:
+        analytics.log(inCallEvents.AUDIO_MUTE, { muting: expectedState });
+        break;
+    case requestTypes.VIDEO_MUTE:
+        analytics.log(inCallEvents.VIDEO_MUTE, { muting: expectedState });
+        break;
+    }
+}
```

Comment:
Looking at amplitude the user timeline looks clear enough to you? I'm a little worried about the timeline getting too noisy but I'll trust your judgement.

Reply:
I think it's okay so far. I'm really trying to implement events from your other PR, so Im thinking you were kind of okay with logging audio and video mute.

Reply:
My comment shouldn’t have been done on a specific line. (It’s a habit of mine to leave the comment whenever I think of it.) I was asking in general and not about mute specifically, but the answer would still be it looks okay.